### ISSUE TITLE
Plain sql as IQueryable

### ISSUE THREAD

**Issue by Franki1986** [linq2db/linq2db#1022]
Hello,

is ist possible to write a Query<T>(string) and return a IQueryable istead of executing it directly?

```
        [Table("CUSTOMER")]
	public partial class Customer
	{
		[Column("ID"), PrimaryKey,  NotNull] public int    Id { get; set; } 
		[Column(), NotNull] public string Name { get; set; } 
		[Column(),  NotNull] public string FirstName { get; set; } 
		[Column(), Nullable         ] public string Company { get; set; } 
		[Column(),  NotNull] public string ZipCode { get; set; }
		[Column(), NotNull] public string Town { get; set; }
	}

       using(var dbContext = new MyDbContext()){
          var customers = dbContext.Query<Customer>("select c.* from CUSTOMER").Where(c => c.FirstName != null);
          var result = customers.ToList();
       }
```

**Comment by sdanyliv** [comment]
Not yet. We are working on this together with CTE feature. It needs some refactoring in internal structures.

**Comment by Franki1986** [comment]
Ok, thanks for reply.. Will close it as not implemented yet. By the way if you have time you could look at calculated fields :-)

**Comment by sdanyliv** [comment]
As i know whole team is under high load this and maybe next month on primary job. So we are trying to finish started features when we can find free time :(

**Comment by Franki1986** [comment]
Id there any update to this?

**Comment by sdanyliv** [comment]
@Franki1986, will handle this.

**Comment by MaceWindu** [comment]
It is something like EF [FromSql()](https://docs.microsoft.com/en-us/ef/core/querying/raw-sql)?

**Comment by sdanyliv** [comment]
Yes. But EF still needs entity configuring. You can do that only with entities in model.

**Comment by MaceWindu** [comment]
closed by #1388
